CODEOWNERS

```diff
@@ -0,0 +1,31 @@
+# CODEOWNERS for autoreview assigning in github
+
+# Order is important; the last matching pattern takes the most
+# precedence.
+
+# Root folder
+/CODEOWNERS                               @carlescufi
+/LICENSE                                  @carlescufi
+/Jenkinsfile                              @carlescufi
+/README.rst                               @ru-fu @carlescufi
+
+# All cmake related files
+/CMakeLists.txt                           @SebastianBoe
+
+# All Kconfig related files
+Kconfig*                                  @SebastianBoe
+
+# All documentation related files
+*.rst                                     @ru-fu
```

Comment:
Put b-gent here as default (you probably don't need the exceptions anymore then).

FYI @b-gent